src/CreeDictionary/API/search/eip_crk.py

```diff
@@ -0,0 +1,140 @@
+# Cree-specific data for English Inflected Phrase search
+from morphodict.analysis.tag_map import TagMap
+
+
+def get_noun_tags(inflectional_category):
+    """Turn a wordform’s inflectional class into tags needed for FST generator.
+
+    >>> get_noun_tags("NA")
+    '+N+A'
+    >>> get_noun_tags("NA-3")
+    '+N+A'
+    >>> get_noun_tags("ND")
+    '+N+D'
+    """
+    noun_tags = ""
+    for c in inflectional_category:
+        if c == "-":
+            return noun_tags
+        noun_tags += "+" + c
```

Comment:
This makes some strong assumptions about the inflectional categories provided in the dictionary database. How likely is it that this won't spit out junk data, like if I accidentally pass it `IPV`, what would happen? How likely is this to be an issue?

Reply:
[This](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/4d9e9ca058c24cb40a375103da3b22b4006a90b2/src/CreeDictionary/API/search/eip.py#L160) is the only place `get_noun_tags` is ever called, so a tag like `IPV` should never make it there.

Reply:
_Should_ never make it there is different than _will never_ make it there 😱 

I have paranoid suggestions of how to improve this, but frankly, those improvements can wait 😅